libbeat/processors/add_host_metadata/add_host_metadata_test.go

```diff
@@ -31,4 +34,39 @@ func TestRun(t *testing.T) {
 	v, err := newEvent.GetValue("host.os.family")
 	assert.NoError(t, err)
 	assert.NotNil(t, v)
+
+	v, err = newEvent.GetValue("host.ip")
+	assert.Error(t, err)
+	assert.Nil(t, v)
+
+	v, err = newEvent.GetValue("host.mac")
+	assert.Error(t, err)
+	assert.Nil(t, v)
+
+	event = &beat.Event{
```

Comment:
Could you split this up into 2 tests? Like this if the tests fail we can already tell based on the name which part is probably not working.

Reply:
@ruflin Is it really enough to `make update` in the metricbeat directory? Or should I do it for every beat?

Reply:
Sorry, my mistake. You have to run it in the top level `beats` directory. Same for `make fmt`.